### PR TITLE
TP-719 Edit footnote description

### PR DESCRIPTION
**Pull request by DIT-CJWaszczuk**
## Summary 
Allow the ability to edit footnotes
### List page 
image
### Edit page 
image
### Confirmation page 
image
### Tariff changes list
image

### CODE REVIEW

**Comment by stuaxo** [comment]
I guess this is test data that's quite long, or would it be worth expanding the text field beyond one line ?

**Comment by DIT-CJWaszczuk** [comment]
Good spot, it should be a textarea (also is in the design)

File: footnotes/urls.py [hunk 1]
```diff
@@ -36,20 +37,14 @@
         name="footnote-ui-confirm-update",
     ),
     re_path(
-        (
-            description_detail
-            + r"description/(?P<description_period_sid>[0-9]{1,8})/edit/$"
-        ),
-        views.FootnoteDescriptionUpdate.as_view(),
-        name="footnote-ui-description-edit",
+        fr"{description_detail}/description/{DESCRIPTION_ID_PATTERN}/edit/$",
```
Comment (andyhd) [inline footnotes/urls.py:40]: To do a custom path converter for footnotes (and other models with compound identifiers) will be a bit more involved, so best leave it for later.

**Review by andyhd — APPROVED** [review]
Looks good, although I think there is a stronger argument for implementing custom path converters now - but I've created [a ticket for it](https://uktrade.atlassian.net/browse/TP-724), so maybe ignore me for this PR at least

**Comment by DIT-CJWaszczuk** [comment]
Changed the description textbox to a textarea and added the details opener with formatting options. Think this is gtg now

**Review by stuaxo — APPROVED** [review]

**Pull request merged**